apps/main/src/utils/parseLocaleNumber.ts

```diff
@@ -0,0 +1,7 @@
+export function parseLocaleNumber(value: string | number | undefined | null): number | null {
```

Comment:
Is this really something that wasn't necessary before? How did we deal with this?
Anyway it doesn't look app-specific, it should be in curve-ui-kit

NB: Is this really supposed to be in `apps/main`? No other files in this app are changed :thinking: